core-build/gulp-core-build/package.json

```diff
@@ -17,7 +17,6 @@
   "dependencies": {
     "@types/assertion-error": "1.0.30",
     "@types/chai": "3.4.34",
-    "@types/chalk": "0.4.31",
```

Comment:
Technically there should be no need to.. I ran "rush install" and it was happy. The next time someone runs generate it will remove the old dependency. This was by-design as Pete and I figured that the extra storage/install time in having the extra dependency sitting around wasn't worth forcing devs to run "generate" (potentially picking up other breaks, causing merge conflicts, etc).